### PR TITLE
Document UI

### PR DESCRIPTION
**Pull request by jreynard-code**
- DocZ is in place.
- Basic MDX files are created with a basic usage of component
- EvolutionText component has been removed
- KPICard has been removed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
-# webapp-component-ui
+# WebApp Component UI
+
+The ***WebApp Component UI*** aims to be a library of [React](https://reactjs.org/) components that can be used for customize/enhance our [Azure Sample Webapp](https://github.com/Cosmo-Tech/azure-sample-webapp).
```
Comment (csm-thu) [inline README.md:3]: _that can be used **to*** customize/enhance_

File: README.md [hunk 2]
```diff
@@ -1 +1,30 @@
...
+
+This project documents and shows how to use the components.
+
+You can use this project as a component library to build a front-end for your own Digital Twin solution.
+
+Feel free to contribute to this library. 
+More we have components, easier your webapp customization will be.
```
Comment (csm-thu) [inline README.md:10]: _The more components we have, the easier your webapp customization will be._

File: src/UploadFile/UploadFile.js [hunk 3]
```diff
@@ -92,17 +91,70 @@ const UploadFile = (props) => {
...
   file: PropTypes.object,
+  /**
+   *  Define the UploadFile's state:
+   *  - true : the button is enabled
+   *  - false : the button is disabled
+   */
   editMode: PropTypes.bool.isRequired,
-  isFileValid: PropTypes.bool
+  /**
+   *  Define the uploaded file is valid or not
```
Comment (FMO-Cosmo) [inline src/UploadFile/UploadFile.js:132]: Defines if the uploaded file is valid or not

File: src/UserInfo/UserInfo.mdx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+route: /UserInfo
+menu: Menus
+---
+import { Playground, Props } from 'docz'
+import { UserInfo } from './UserInfo'
+import profilePic from '../../assets/profile_placeholder.png'
+
+# User Information
+
+The aim of this component is to easily display a  user information
```
Comment (FMO-Cosmo) [inline src/UserInfo/UserInfo.mdx:12]: easily display user information

**Review by FMO-Cosmo — CHANGES_REQUESTED** [review]
Nice documentation. Congrats for all the work !
(just small corrections to apply before merge)

File: src/SimpleTwoActionsDialog/SimpleTwoActionsDialog.js [hunk 5]
```diff
@@ -13,63 +12,109 @@ import {
...
           color="primary">
-          {t(cancelLabelKey, 'Cancel label')}
+          {labels.button1}
         </Button>
         <Button
-          data-cy='dialog-discard-button'
-          id="ButtonDiscard"
-          onClick={handleClickOnValidate}
+          data-cy={id + '-button2'}
+          id={id + 'id-button2'}
```
Comment (csm-thu) [inline src/SimpleTwoActionsDialog/SimpleTwoActionsDialog.js:49]: The id of _button2_ is not consistent with `id={id + '-button1'}` for _button1_.
Also _button2_ has a data-cy prop but _button1_ does not.

**Review by csm-thu — CHANGES_REQUESTED** [review]
Some suggestions for improvements. 
Great work though :+1:

**Review by csm-thu — APPROVED** [review]

**Pull request merged**